Persephone/Components/Browser/Album Detail/AlbumDetailView.swift

```diff
@@ -132,20 +128,23 @@ class AlbumDetailView: NSViewController {
         self.dataSource.albumSongs[1].song
         else { return }
 
-      self.getBigCoverArt(song: song)
+      self.getBigCoverArt(song: song, album: album)
     }
   }
 
-  func getBigCoverArt(song: Song) {
-    let coverArtService = CoverArtService(song: song)
-
-    coverArtService.fetchBigCoverArt()
-      .done(on: DispatchQueue.main) { [weak self] image in
-        if let image = image {
-          self?.albumCoverView.image = image
-        }
-      }
-      .cauterize()
+  func getBigCoverArt(song: Song, album: Album) {
+    guard let imagePath = album.coverArtFilePath else { return }
+
+    let imageURL = URL(fileURLWithPath: imagePath)
+    let provider = LocalFileImageDataProvider(fileURL: imageURL)
+    albumCoverView.kf.setImage(
+      with: .provider(provider),
+      placeholder: NSImage.defaultCoverArt,
+      options: [
+        .processor(DownsamplingImageProcessor(size: NSSize(width: 500, height: 500))),
```

Comment:
Would it be helpful if the `NSSize()` values were a constant or an enum somewhere? I'm thinking something like `Size.medium` or `Size.coverArt`.